src/main/java/core/basesyntax/db/impl/StorageImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.db.impl;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl implements Storage {
+    private final Map<String, Integer> storage = new HashMap<>();
+
+    @Override
+    public Map<String, Integer> getStorage() {
+        return storage;

```

Comment:
Directly returning the map from the getStorage method can lead to potential data leakage or accidental modification as it violates the principle of data encapsulation. Consider returning a copy of the map or an unmodifiable map view instead.